### PR TITLE
SC-6682 - decreases search time in lern-store

### PR DESCRIPTION
**Pull request by JenyMzo**
# Description

Improves search interaction in lern-store search.

## Links to Tickets or other pull requests

https://ticketsystem.hpi-schul-cloud.org/browse/SC-6682

## Changes

- Shows the spinner everytime the search parameter changes.
- Adds a hint to the user asking to type at least 2 characters to perform the search.
- Decreases the time in which the spinner shows up after typing.

## Screenshots of UI changes

Screenshot 2020-09-18 at 10 17 02

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Comment by murat-merdoglu-hpi** [comment]
@schul-cloud-bot update screenshots

**Comment by mwolowyk-cap** [comment]
@schul-cloud-bot update screenshots

File: src/pages/content/index.vue [hunk 1]
```diff
@@ -132,25 +136,28 @@ export default {
 			if (this.$options.debounce) {
 				clearInterval(this.$options.debounce);
 			}
+
 			if (to === from || !to) {
 				this.$router.push({
 					query: {
 						...this.$route.query,
-						q: undefined,
+						q: "",
```
Comment (mwolowyk-cap) [inline src/pages/content/index.vue:144]: try to remove the q parameter. We don't want to have search parameter if there is no search term.

**Review by mwolowyk-cap — CHANGES_REQUESTED** [review]
please add your changes to Changelog.md

**Review by mwolowyk-cap — APPROVED** [review]

**Comment by JenyMzo** [comment]
@schul-cloud-bot update screenshots

**Review by mwolowyk-cap — APPROVED** [review]

**Pull request merged**